src/ui/pages/Menu/components/Settings/Settings.tsx

```diff
@@ -78,9 +79,13 @@ const Settings = () => {
     switch (item.label) {
     case i18n.t("settings.sections.security.biometry"): {
       setBiometricsIsEnabled(!biometricsIsEnabled);
-      await PreferencesStorage.set(PreferencesKeys.APP_BIOMETRY, {
-        enabled: !biometricsIsEnabled,
-      });
+      await createOrUpdateBasicRecord(
```

Comment:
Note here that it would be better if this just updated, since once it's set, it's done for good. But I think it's worth the simplicity and stability of not assuming.

If we find that it's too slow when enabling/disabling the biometrics very quickly - we can re-visit and see. Maybe can also have a cache write back policy or something.